build/nuget-release.yml

```diff
@@ -0,0 +1,172 @@
+            inputs:
+              command: 'custom'
+              custom: 'new'
+              arguments: '-i "templates/*.nupkg"'
+          - task: DotNetCoreCLI@2
+            displayName: 'Create new project from template'
+            inputs:
+              command: 'custom'
+              custom: 'new'
+              arguments: 'arcus-webapi -n MyProject -o /MyProject'
+          - task: Docker@1
+            displayName: 'Build Docker image from newly created web API project from template'
+            inputs:
+              dockerFile: /MyProject/Dockerfile
+              imageName: 'myproject:$(Build.BuildId)'
+          - task: Docker@1
+            displayName: 'Run new web API project Docker image from template'
+            inputs:
+              command: 'Run an image'
+              imageName: 'myproject:$(Build.BuildId)'
+              containerName: 'myproject'
+              ports: '$(Http.Port):$(Http.Port)'
+              envVars: |
+                ARCUS_HTTP_PORT=$(Http.Port)
+          - task: qetza.replacetokens.replacetokens-task.replacetokens@3
+            displayName: 'Replace integration test tokens'
+            inputs:
+              rootDirectory: 'src/$(Project).Tests.Integration/'
+              targetFiles: 'appsettings.json'
+              encoding: 'auto'
+              writeBOM: true
+              actionOnMissing: 'fail'
+              keepToken: false
+              tokenPrefix: '#{'
+              tokenSuffix: '}#'
+          - task: DotNetCoreCLI@2
+            displayName: 'Run integration tests'
+            inputs:
+              command: test
+              projects: 'src/**/$(Project).Tests.Integration.csproj'
+              arguments: '--configuration $(Build.Configuration)'
+              nobuild: true
+              publishTestResults: true
+
+  - stage: Release
+    displayName: 'Release to NuGet.org'
+    dependsOn: IntegrationTests
+    condition: succeeded()
+    jobs:
+      - job: PushToNuGet
+        displayName: 'Push to NuGet.org'
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DownloadPipelineArtifact@2
+            displayName: 'Download build artifacts'
+            inputs:
+              artifact: 'Templates'
+              path: '$(Build.SourcesDirectory)/templates'
+          - task: GitHubRelease@0
+            displayName: 'Create GitHub Release (Conditional Step)'
+            inputs:
+              gitHubConnection: 'GitHub (arcus-automation - OAuth)'
+              repositoryName: 'arcus-azure/arcus.templates'
+              tagSource: manual
+              tag: 'v$(Build.BuildNumber)'
+              title: 'v$(Build.BuildNumber)'
+              releaseNotesSource: input
+              releaseNotes: |
+                Install new version via [NuGet](https://www.nuget.org/packages/Arcus.Template.WebApi/$(Build.BuildNumber))
+                ```shell
+                > Install-Package Arcus.Template.WebApi -Version $(Build.BuildNumber)
```

Comment:
If the `VersionSuffix` is set (via the YAML template) and not the `Version`; what should this be?

Reply:
Nevermind; the suffix is used to set the version.